docs/data-sources/aws_crossaccount_policy.md

```diff
@@ -3,10 +3,10 @@ subcategory: "Deployment"
 ---
 # databricks_aws_crossaccount_policy Data Source
 
--> **Note** This data source can only be used with an account-level provider!
-
 This data source constructs necessary AWS cross-account policy for you, which is based on [official documentation](https://docs.databricks.com/administration-guide/account-api/iam-role.html#language-Your%C2%A0VPC,%C2%A0default).
 
+-> This data source can be used with an account or workspace-level provider.
```

Comment:
```suggestion
-> This data source can be used with an account-level provider.
```
this data source only makes sense for account-level context

Reply:
Technically it works with both: https://github.com/databricks/terraform-provider-databricks/blob/main/aws/data_aws_crossaccount_policy.go#L26 - we changed it to account only, and it broke people's work: https://github.com/databricks/terraform-provider-databricks/issues/3505